### ISSUE TITLE
Loss always remains nan 

### ISSUE THREAD

**Issue by afaq-ahmad** [tztztztztz/faster_rcnn.pytorch#2]
I have started to train the model on Voc 2007 dataset, without any pre-trained model but loss not decreasing. Any thoughts on it that how can we solve it?
[epoch  1][step   50/10022] loss: nan, lr: 1.00e-03, time cost 92.7s
                         rpn_cls_loss_v: nan, rpn_box_loss_v: nan
                         rcnn_cls_loss_v: nan, rcnn_box_loss_v: nan

**Comment by tztztztztz** [comment]
Please use a pre-trained model.

**Comment by afaq-ahmad** [comment]
So In my case i am finding specific patterns in text, so can;t use imagenet,coco etc pretrained model

**Comment by tztztztztz** [comment]
What do you mean by "finding specific patterns in text"?

You said you train the model with VOC 2007, so I think you could always use a pretrained model. for example, ResNet50: https://drive.google.com/open?id=0B7fNdx_jAqhtbllXbWxMVEdZclE

**Comment by afaq-ahmad** [comment]
Thanks for your kind reply,

So My annotations are in voc xml format. I am finding specific marking fields in scan documents, I have trained tensorflow object detection API fasterrcnn and its performing good. But now I want to change little bit fasterrcnn architecture (adding some attention based on computer vision techniques), So far that I have to first run a faster rcnn implementation. 

I have tried on my own dataset but loss is nan. And even tried on original voc dataset but loss is nan. So my question is why this problem appearing and model should be trainable without pretrained weights. is it because of weights initialization? Should I work on it?  Any of your guidelines would quite helpful. thanks

**Comment by tztztztztz** [comment]
If you want your model can be trained without pretrained weights, you may lower your learning rate and increase batch size to stable the training process.

FYI, the tensorflow detection API uses pretrained model too.

And I have a question, why don't you use pretrained model. What is your backbone?

**Comment by afaq-ahmad** [comment]
@tztztztztz Thanks
